matrix_is_tester/fakehs.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm afraid I'm struggling to grok what forms the public interface to this file. Where do I start, when I am trying to use it?

Any or all of the following might help:

- ordering the file consistently (put the highest-level stuff at the top?)
- docstrings
- give private stuff an underscore prefix